evm/src/lib.rs

```diff
@@ -2,8 +2,10 @@
 #![allow(clippy::too_many_arguments)]
 #![allow(clippy::type_complexity)]
 #![feature(generic_const_exprs)]
+#![feature(array_zip)]
 
 pub mod all_stark;
+pub mod alu;
```

Comment:
Sorry for all the pedantic comments 😄 but now that we moved logic to another table, maybe `arithmetic` would be a good name for this one